core/client/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -148,11 +148,11 @@ public static String toStringHadoopInputSplit(InputSplit is) {
    * @param conf Hadoop configuration
    */
   public static void addS3Credentials(Configuration conf) {
```

Comment:
+1, to track this: https://alluxio.atlassian.net/browse/ALLUXIO-2181

I'll create a PR after this one got merged.